templates/helm/values.yaml.tpl

```diff
@@ -27,6 +27,12 @@ resources:
 aws:
   # If specified, use the AWS region for AWS API calls
   region: ""
+  account_id: ""
+
+# log level for the controller
+log:
+  enable_development_logging: true
+  log_level: info
```

Comment:
`DEBUG` did not work, hence I kept it as `info` and not `INFO`

Reply:
The `./kind-build-test.sh` script has this set to `debug`, but I don't think that's appropriate for production use anyway. I think this is good.

Reply:
Actually, I am inclined to keep default as debug because info logs are very less and hard to relate with state of resource. I think even the delta is not printed in info. Let's see what others have to say.

Reply:
> Actually, I am inclined to keep default as debug because info logs are very less and hard to relate with state of resource. I think even the delta is not printed in info. Let's see what others have to say.

Delta is shown in Info, actually: https://github.com/aws-controllers-k8s/runtime/blob/main/pkg/runtime/reconciler.go#L251

I would prefer to use info level for default.

Reply:
@surajkota is correct. We are logging the diff/delta using V(1) (debug) not info...

That said, I would prefer to merge this with info as the default log level and address specific log messages (and moving some to info vs debug, etc) in a separate PR.

Reply:
Makes sense, I can create a separate issue/pr for that